### ISSUE TITLE
The learning rate, loss function and optimization function in experiment.

### ISSUE THREAD

**Issue by Infinite-Song** [hyperfraise/Pytorch-StNet#4]
Hi, I trained this model on UCF-101 from scratch.  The learning rate was set as 0.001. I chose the cross entropy as the loss function and SGD as the optimization function. However, after finishing training, the result  in validation set is only about 60%, which seems not correspond to the paper.  

**Could you please tell me these parameters in your experiments? Thanks!**

**Comment by dmenig** [comment]
On the paper, no experiments results are shown concerning UCF-101 from scratch. The results available for this dataset are all obtained through transfer from kinetics. So maybe try transferring from kinetics and you'll be able to reproduce the results from the paper. But as I said, I'm not among the authors of the article, and thus couldn't tell you more than what is outlined in it, about the experimental setup leading to the results shown.

**Comment by zkqiu** [comment]
You might encounter overfitting problem
